drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -239,10 +244,10 @@ function DragAndDropTemplates(configuration) {
         });
 
         return (
-            h('section.feedback', {}, [
+            h('div.feedback', {attributes: {'role': 'group', 'aria-label': gettext('Feedback')}}, [
                 h(
                     "div.feedback-content",
-                    { attributes: { 'aria-live': 'polite' } },
+                    {attributes: {'aria-live': 'polite', 'aria-atomic': 'true', 'aria-relevant': 'all'}},
```

Comment:
If the aria-live functionality on this feedback-content div is not working, we should remove it. As it is, if it should happen to work wouldn't the user be inundated with double messaging due to the usage of SR.readText?